### ISSUE TITLE
[Accessibility][CreateDashboardSuccessNotification] Decorative images are not hidden from screen reader users

### ISSUE THREAD

**Issue by mnischay** [awslabs/iot-application#1841]
#### Notes and Advice 
The white tick mark image is a decorative image that is not hidden from screen reader users. Images that are decorative and do not convey any meaningful information to users must be hidden from assistive technologies.
 Recommendation 
Use an empty string as text alternative aria-hidden="true". This will ensure that the image is not announced by assistive technologies.
 
#### Recommended (simplified) markup
 
<div role="img" aria-hidden="true">
 
In HTML, empty attributes have an implicit null value, so alt has the same effect as alt="".

 ####Resources 
[Decorative Images](https://www.w3.org/WAI/tutorials/images/decorative/)

### AC

- [ ] The above violation is mitigated

**Comment by kausar-shaikh** [comment]
![Screenshot from 2023-12-12 12-04-55](https://github.com/awslabs/iot-application/assets/126452015/56d1a371-7f97-4f76-9b4b-e89b50dab559)

**Comment by kausar-shaikh** [comment]
[Screencast from 12-12-23 12:06:58 PM IST.webm](https://github.com/awslabs/iot-application/assets/126452015/ca42975a-8f36-4b56-a449-bc0bc6dbb0c1)

**Comment by Kausar-HM** [comment]
The screen reader is reading out "success" image unless we remove 'successIconAreaLabel' from 'i18nStrings' passed to Flashbar, but that introduces another accessibility issue.

We also checked the screen reader behaviour is same for this decorative icon in cloudscape demo page. 
We do not have any way(to pass props) to hide the status icons on the <Flashbar /> component to hide the decorative images from screen reader users.

**Comment by jmbuss** [comment]
This looks like a cloudscape concern. Let's pause work on this while I confirm the next steps with them.

**Comment by fpauer** [comment]
This repo is deprecated, so plan to fix this issue.
